### PR TITLE
[OFFAPPS-1172] Better error when ticket unlinked

### PR DESCRIPTION
**Pull request by ocke**
## Description
We add a small but neat feature that displays a warning when a child ticket can't be found. It additionally gives one the option to remove the link, so they can relink the ticket.

Also resize the app every time we `switchTo`

Also only make api call if we need the info, so we don't make unnecessary api calls.

**Go by commit for easy reviewing**

## References
[JIRA ticket](https://zendesk.atlassian.net/browse/OFFAPPS-1172)

## CCs
@zendesk/apps-migration
@zendesk/vegemite 

## Risks
* [low] Linked ticket app gives an error when unlinking a ticket

### CODE REVIEW

File: src/javascript/lib/view.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ class View {
   switchTo (name, data) {
     $('[data-main]').html(this.renderTemplate(name, data))
     $.isFunction(this.afterRender) && this.afterRender()
+    this.resizeIframe()
```
Comment (keitheous) [inline src/javascript/lib/view.js]: Hello Olaf, curious about invoke height and width defined on [line 22](https://github.com/zendesk/linked_ticket_app/blob/3400a238b4ab518bfeee73d2dec71718b1851b84/src/javascript/lib/view.js#L22), do you think we should do one first before another to work around how zaf resize?

File: src/javascript/lib/view.js [hunk 2]
```diff
@@ -14,6 +15,11 @@ class View {
     $('[data-main]').html(this.renderTemplate(name, data))
     $.isFunction(this.afterRender) && this.afterRender()
   }
+
+  resizeIframe () {
+    const height = document.querySelector('body').scrollHeight
+    client.invoke('resize', { width: '100%', height: height })
```
Comment (keitheous) [inline src/javascript/lib/view.js]: Hello Olaf, should we do one first before another to work around how zaf resizes app?

File: src/javascript/app.js [hunk 3]
```diff
@@ -134,23 +134,22 @@ const App = {
...
     }).then((data) => {
-      const [ticketFieldData, ancestryField, ancestryFieldId] = data
-      const ticketField = ticketFieldData[`ticketFields:custom_field_${ancestryFieldId}`]
-
-      if (!ticketField) {
-        const errorMessage = this.I18n.t('linked_data_field_missing_this_form', {
-          field_name: ancestryField.title,
-          field_id: ancestryField.id
+      const [ancestryFieldId, hasTicketField] = data
+      console.log(hasTicketField)
```
Comment (keitheous) [inline src/javascript/app.js]: Console.log

Comment (ocke) [reply src/javascript/app.js]: oops

**Pull request merged**